source/mongorestore/mongorestore-examples.txt

```diff
@@ -345,14 +345,27 @@ namespace:
        You cannot restore the ``system.buckets.weather.bson`` file
        by itself. Attempting to do so results in an error.
 
+Back Up and Restore a Database at a Specific Point in Time
+----------------------------------------------------------
+
+To back up a database at a specific point in time to an :term:`oplog`
```

Comment:
I think part of the confusion here is people assume they have control over the specific timestamp to use as the point in time.
It's no longer safe to attempt to choose a timestamp on which to stop mongoreplay-ing. IMO it's better to brand --oplog mode as a means to avoid pausing writes for the full duration of the backup rather than a point in time recovery service.
We could instead label this example "mongorestore --oplogReplay and mongodump --oplog" or something similar.

Reply:
I'll flag this to some folks from the DB Tools team for input, since it looks like I can't CC them directly on this PR comment.